docs/en/stack/ml/df-analytics/dfa-classification.asciidoc

```diff
@@ -65,6 +65,24 @@ have a similar number of data points for each class.
 ////
 
 
+[[dfa-classification-feature-encoding]]
+=== Feature encoding
+
+Every {ml} model can only work with numerical values. For this reason, it is 
+required to transform the categorical values of the relevant features into 
+numerical ones. This process is called feature encoding.
+
+{dfanalytics-cap} automatically performs feature encoding. During which the 
+input data is pre-processed with the following encoding techniques:
+
+* one-hot encoding,
+* target-mean encoding,
+* frequency encoding.
+
+After the {classification} model performed the prediction, {dfanalytics} maps 
+the numerical output provided by the model to the categorical labels.
```

Comment:
I don't understand what you are trying to say here. Here is some more information that you might want to include.

When the model makes predictions on new data, the data needs processing in the same way in which it is trained. 

Machine learning model inference in the elastic stack does this automatically. Meaning, the automatically applied encodings are used in each call for inference.

When feature importance is calculated, we calculate the feature importance for the original categorical fields. Not the automatically encoded features.